browsergym/core/src/browsergym/core/action/functions.py

```diff
@@ -82,8 +88,15 @@ def check(bid: str):
         check('55')
     """
     elem = get_elem_by_bid(page, bid, demo_mode != "off")
-    add_demo_mode_effects(page, elem, bid, demo_mode=demo_mode, move_cursor=True)
-    elem.check(timeout=500)
+    if demo_mode != "off":
```

Comment:
Not needed, a check is done inside the function
https://github.com/ServiceNow/BrowserGym/blob/b21860d0df4a3d5396ac5afcd5cfdf61bd9d341a/browsergym/core/src/browsergym/core/action/utils.py#L244

Reply:
oh! ok will revert